docs/DOCKER.rst

```diff
@@ -1,43 +1,35 @@
 Docker
 ======
 
--  `For continuous integration with
-   Docker <#for-continuous-integration-with-docker>`__
 -  `Environment Variables <#environment-variables>`__
--  `In production with Docker <#in-production-with-docker>`__
--  `Using Docker Volume in
-   production <#using-docker-volume-in-production>`__
--  `Adding modifying or deleting accounts or users
-   credentials <#adding-modifying-or-deleting-accounts-or-users-credentials>`__
--  `Specifying your own host name <#specifying-your-own-host-name>`__
--  `Running as an unprivileged
-   user <#running-as-an-unprivileged-user>`__
-
-For continuous integration with Docker
---------------------------------------
-
-When you start the Docker Scality Zenko CloudServer image, you can adjust the
-configuration of the Scality Zenko CloudServer instance by passing one or more
-environment variables on the docker run command line.
+-  `Tunables and setup tips <#tunables-and-setup-tips>`__
+-  `Examples for continuous integration with
+   Docker <#continuous-integration-with-docker-hosted CloudServer>`__
+-  `Examples for going in production with Docker <#in-production-with-docker-hosted CloudServer>`__
 
 Environment Variables
-~~~~~~~~~~~~~~~~~~~~~
-
-S3DATA=multiple
-^^^^^^^^^^^^^^^
+---------------------
 
-This runs Scality Zenko CloudServer with multiple data backends. `More
-info <../GETTING_STARTED/#location-configuration>`__
+S3DATA
+~~~~~~
 
-.. code:: shell
+S3DATA=multiple and S3DATA=scality
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Either of these runs Scality Zenko CloudServer with multiple data backends. The
+scality value refers to installations with a RING as one of the backends.
+When using multiple data backends, a custom locationConfig.json file is
+mandatory. It will provide custom regions and associated rest_endpoints
+configuration `More info <../GETTING_STARTED/#location-configuration>`__
 
-    docker run -d --name s3server -p 8000:8000 -e S3DATA=multiple scality/s3server
+If you are using RING endpoints, please refer to your customer documentation.
```

Comment:
I don't think we should have this here.

Reply:
We do have a few customers who check the open source documentation. Are you strongly against it, or can we keep it?

Reply:
At least Scality RING then.

Reply:
Changed, thanks